sktime/forecasting/compose/_reduce.py

```diff
@@ -1885,10 +1889,24 @@ class DirectReductionForecaster(BaseForecaster, _ReducerMixin):
         "panel" = second lowest level, one reduced model per panel level (-2)
         if there are 2 or less levels, "global" and "panel" result in the same
         if there is only 1 level (single time series), all three settings agree
+    windows_identical : bool, optional, default=True
+        Specifies whether all direct models use the same number of observations
+        or a different number of observations.
+            * `True`: Uniform window of length (total observations - maximum
```

Comment:
I think the bullet point shoud be with space and one indent less.

That is,

```
Specifies whether

* ` True` : etc
```

Reply:
Done.